src/app/components/App.js

```diff
@@ -13,7 +14,8 @@ export default class App extends Component {
     <Suspense fallback={<Loading>Loading</Loading>}>
       <Switch>
         <Route exact path="/" render={Screen(LandingScreen)}/>
-        <Route exact path="/article/:id" render={Screen(ArticleScreen)}/>
+        <Route exact path="/article/old" render={Screen(ArticleScreen)}/>
+        <Route exact path="/article/microplastic" render={Screen(MicroPlasticScreen)}/>
         <Route render={Screen(NotFoundScreen)}/>
```

Comment:
The article screen should be a template and the `:id` param should be used to dynamically change the content from a config file. What are you doing here adding a /old and /microplastic route? This seems very wrong.